tests/driver_soft_spi/Makefile

```diff
@@ -0,0 +1,17 @@
+APPLICATION = driver_soft_spi
+include ../Makefile.tests_common
+
+BOARD ?= stm32f4discovery
+
+BOARD_WHITELIST := stm32f4discovery samr21-xpro
+
+USEMODULE += soft_spi
+USEMODULE += periph_common
```

Comment:
better: add `periph_common` as dependency to `soft_spi` in `drivers/Makefile.dep`!

Reply:
✓